.github/workflows/build_test_deploy.yml

```
@@ -241,6 +241,52 @@ jobs:
         name: Run test/development
         if: ${{needs.build.outputs.docsChange != 'docs only change'}}
 
+      - name: Upload test trace
+        if: always()
+        uses: actions/upload-artifact@v2
+        with:
+          name: test-trace
+          if-no-files-found: ignore
+          retention-days: 2
+          path: |
+            test/traces
+
+  testDevE2E:
+    name: Test Development (E2E)
+    runs-on: ubuntu-latest
+    needs: [build, build-native-dev]
+    env:
+      NEXT_TELEMETRY_DISABLED: 1
+      NEXT_TEST_JOB: 1
+    steps:
+      - name: Setup node
+        uses: actions/setup-node@v2
+        if: ${{ steps.docs-change.outputs.docsChange != 'docs only change' }}
```

```suggestion
        if: ${{needs.build.outputs.docsChange != 'docs only change'}}
```

Is there a typo here?

---

Yeah looks like the setup node docs checks are wrong, updated in https://github.com/vercel/next.js/pull/34827/commits/94d05290fd039e6254412f926847bf15165d3723